inc/az_ulib_descriptor_api.h

```diff
@@ -128,79 +128,79 @@ typedef struct az_ulib_interface_descriptor_tag {
+ * command will conclude the operation firing a callback with the result.
  *
  * The hardware operation may be cancellable, so, the API allows the developer to provide a cancel
  * function.
  *
- * @param[in]   name            The `/0` terminated `const char* const` with the method name. It
+ * @param[in]   name            The `/0` terminated `const char* const` with the command name. It
  *                              cannot be `NULL` and shall be allocated in a way that it stays
  *                              valid until the interface is unpublished at some (potentially)
  *                              unknown time in the future.
- * @param[in]   method_async    The function pointer to #az_ulib_capability_method_async with the
- *                              implementation of the asynchronous method. The method shall be
+ * @param[in]   command_async    The function pointer to #az_ulib_capability_command_async with the
+ *                              implementation of the asynchronous command. The command shall be
  *                              valid until the interface is unpublished at some (potentially)
  *                              unknown time in the future.
  * @param[in]   cancel          The function pointer to #az_ulib_capability_cancellation_callback
  *                              with the implementation of the function to cancel the asynchronous
- *                              method. It can be `NULL` if the method does not allow any
+ *                              command. It can be `NULL` if the command does not allow any
  *                              cancellation. If provided, the cancel shall be valid until the
  *                              interface is unpublished at some (potentially) unknown time in the
  *                              future.
- * @return The #az_ulib_capability_descriptor with the method.
+ * @return The #az_ulib_capability_descriptor with the command async.
  */
-#define AZ_ULIB_DESCRIPTOR_ADD_METHOD_ASYNC(name, method_async, cancel) \
+#define AZ_ULIB_DESCRIPTOR_ADD_COMMAND_ASYNC(name, command_async, cancel) \
   { \
-    (name), { (const void*)(method_async) }, { (const void*)(cancel) }, \
-        (uint8_t)(AZ_ULIB_CAPABILITY_TYPE_METHOD_ASYNC) \
+    (name), { (const void*)(command_async) }, { (const void*)(cancel) }, \
+        (uint8_t)(AZ_ULIB_CAPABILITY_TYPE_COMMAND_ASYNC) \
   }
 
 /**
- * @brief   Add an event to the interface descriptor.
+ * @brief   Add an telemetry to the interface descriptor.
  *
- * Populate a new *event* capability descriptor to add to the interface. On the interface context, a
- * event is a subscribed base notification, where other components in the system can subscribe to
- * be notified when certain event happened.
+ * Populate a new *telemetry* capability descriptor to add to the interface. On the interface
+ * context, a telemetry is a subscribed base notification, where other components in the system can
```

Comment:
This would be referring to the OS definition of context (thread state) and not DTDL context (DTDL version), correct? May be conflicting language here.

Reply:
Sorry, I do not understand your comment, can you clarify?

---
In reply to: [560626566](https://github.com/Azure/azure-ulib-c/pull/55#discussion_r560626566) [](ancestors = 560626566)

Reply:
It seems as though "context" is used to describe the DTDL version being used when publishing an interface. As is described [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/dtdlv2.md). Just want to make sure this description is consistent with that.

Reply:
I see. No, the word "interface context" here is not related to the DTDL Interface @context. I'll change the sentence to avoid confusion. 

---
In reply to: [561130220](https://github.com/Azure/azure-ulib-c/pull/55#discussion_r561130220) [](ancestors = 561130220)

Reply:
This explanation is not necessary, I removed it and add a link to the capability.

---
In reply to: [561133812](https://github.com/Azure/azure-ulib-c/pull/55#discussion_r561133812) [](ancestors = 561133812,561130220)